includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1922,6 +1961,427 @@ public static function print_status_meta_box( $post ) {
+						}
+						?>
+					</td>
+					<td class="column-final_size">
+						<?php
+						echo esc_html( number_format_i18n( $stylesheet['final_size'] ) );
+						echo '<small>B</small>';
+						?>
+					</td>
+					<td class="column-percentage">
+						<?php
+						$percentage = $stylesheet['final_size'] / ( $included_final_size + $excluded_final_size );
+						?>
+						<meter value="<?php echo esc_attr( $stylesheet['final_size'] ); ?>" min="0" max="<?php echo esc_attr( $included_final_size + $excluded_final_size ); ?>" title="<?php esc_attr_e( 'Stylesheet bytes of total CSS added to page', 'amp' ); ?>">
+							<?php echo esc_html( round( ( $percentage ) * 100 ) ) . '%'; ?>
+						</meter>
+					</td>
+					<td class="column-priority">
+						<?php echo esc_html( $stylesheet['priority'] ); ?>
+					</td>
+					<td class="column-stylesheet_status">
+						<?php
+						switch ( $stylesheet['status'] ) {
+							case $included_status:
+								printf( '<span title="%s">✅</span>', esc_attr__( 'Stylesheet included', 'amp' ) );
+								break;
+							case $excessive_status:
+								printf( '<span title="%s">⚠️</span>', esc_attr__( 'Stylesheet overruns CSS budget yet it is still included on page', 'amp' ) );
+								break;
+							case $excluded_status:
+								printf( '<span title="%s">🚫</span>', esc_attr__( 'Stylesheet excluded due to exceeding CSS budget', 'amp' ) );
+								break;
+						}
+						?>
+					</td>
+					<td class="column-markup">
+						<?php
+						$origin_abbr_text = '?';
+						if ( 'link_element' === $stylesheet['origin'] ) {
+							$origin_abbr_text = '<link&nbsp;&hellip;>'; // @todo Consider adding the basename of the CSS file.
+						} elseif ( 'style_element' === $stylesheet['origin'] ) {
+							$origin_abbr_text = '<style>';
+						} elseif ( 'style_attribute' === $stylesheet['origin'] ) {
+							$origin_abbr_text = 'style="&hellip;"';
+						}
+						$needs_abbr = $origin_abbr_text !== $origin_html;
+						if ( $needs_abbr ) {
+							printf( '<abbr title="%s">', esc_attr( $origin_html ) );
+						}
+						printf( '<code>%s</code>', esc_html( $origin_abbr_text ) );
+						if ( $needs_abbr ) {
+							echo '</abbr>';
+						}
+						echo '</code>';
+						?>
+					</td>
+					<td class="column-sources_with_invalid_output">
+						<?php
+						if ( empty( $stylesheet['sources'] ) ) {
+							esc_html_e( '--', 'amp' );
+						} else {
+							self::render_sources_column( AMP_Validation_Error_Taxonomy::summarize_sources( $stylesheet['sources'] ), $post->ID );
+						}
+						?>
+					</td>
+				</tr>
+				<tr class="<?php echo esc_attr( sprintf( 'stylesheet-details level-0 %s', 0 === $row % 2 ? 'even' : 'odd' ) ); ?>">
+					<td colspan="10">
+						<dl class="detailed">
+							<dt><?php esc_html_e( 'Original Markup', 'amp' ); ?></dt>
+							<dd><code class="stylesheet-origin-markup"><?php echo esc_html( $origin_html ); ?></code></dd>
+
+							<dt><?php esc_html_e( 'Sources', 'amp' ); ?></dt>
```

Comment:
This looks really good:
<img width="756" alt="sources-good" src="https://user-images.githubusercontent.com/4063887/73053740-2d80ac00-3e4e-11ea-8f9f-e33b0def04a0.png">